frontend/src/metabase/forms/components/FormTextInput/FormTextInput.tsx

```diff
@@ -3,15 +3,24 @@ import type { ChangeEvent, FocusEvent, Ref } from "react";
 import { useField } from "formik";
 import { TextInput } from "metabase/ui";
 import type { TextInputProps } from "metabase/ui";
+import { CopyWidgetButton } from "./FormTextInput.styled";
 
 export interface FormTextInputProps
   extends Omit<TextInputProps, "value" | "error"> {
   name: string;
   nullable?: boolean;
+  hasCopyButton?: boolean;
 }
 
 export const FormTextInput = forwardRef(function FormTextInput(
-  { name, nullable, onChange, onBlur, ...props }: FormTextInputProps,
+  {
+    name,
+    nullable,
+    hasCopyButton,
```

Comment:
can you add unit test coverage for this new prop?

Reply:
fd5f11a53413a5656d034192df9356bf53d4e27c